game/scripts/npc/abilities/templar_assassin_refraction.txt

```diff
@@ -23,7 +23,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "17.0"
+    "AbilityCooldown"                                     "17.0 16.0 15.0 14.0 13.0 12.0" // OAA
```

Comment:
Revert this, its already 0 cd downtime.

Reply:
I'd like you to show me an example of it, cause no TA can manage refractions as it's completely up to the competence of the enemy. I'll change it back for now.